tools/go.mod

```diff
@@ -154,12 +153,12 @@ require (
 	github.com/spf13/cobra v1.5.0 // indirect
 	github.com/spf13/jwalterweatherman v1.1.0 // indirect
 	github.com/spf13/pflag v1.0.5 // indirect
-	github.com/spf13/viper v1.12.0 // indirect
+	github.com/spf13/viper v1.13.0 // indirect
```

Comment:
Looks like this is the culprit.  We can add a replace back to v1.12.0 until golangci-lint fixes this behavior.